stix_shifter_modules/lang_en.json

```diff
@@ -5,11 +5,11 @@
         },
         "name": {
             "label": "Data source name",
-            "placeholder": "Add a data source name"
+            "description": "Assign a name to identify the connection minimum of 5 characters"
```

Comment:
You may want to put minimum 5 characters in parenthesis otherwise it looks like a run-on sentence.

Reply:
I think it is a good idea.  I added it.